### PR TITLE
Switch OSX CI innerloop run to use build.sh.

### PR DESCRIPTION
**Pull request by Priya91**
cc @weshaggard @mmitche

### CODE REVIEW

**Comment by Priya91** [comment]
@dotnet-bot test ci please

File: netci.groovy [hunk 1]
```diff
@@ -509,7 +509,7 @@ def innerLoopNonWindowsOSs = ['Ubuntu15.10', 'Debian8.2', 'OSX', 'CentOS7.1', 'O
             Utilities.addArchival(newBuildJob, "bin/build.pack,bin/osGroup.AnyCPU.${configurationGroup}/**,bin/ref/**,bin/packages/**,msbuild.log")
 
             // Use Server GC for Ubuntu/OSX Debug PR build & test
-            def useServerGC = ((os == 'Ubuntu15.10' || os == 'OSX') && configurationGroup == 'Release' && isPR)
+            def useServerGC = ((os == 'Ubuntu15.10') && configurationGroup == 'Release' && isPR)
```
Comment (mellinoe) [inline netci.groovy]: We can probably just remove this entirely from this definition. Originally we were using server GC on Ubuntu14.04 and OSX, and now both of those have moved to the other definition. I had switched it to Ubuntu15.10 as a replacement for Ubuntu14.04, so originally we weren't doing it on Ubuntu15.10 before m change.

**Comment by mellinoe** [comment]
LGTM.

**Comment by mmitche** [comment]
LGTM

File: netci.groovy (hunk 1 above)
Comment (Priya91) [inline netci.groovy]: Ok, will remove.

File: netci.groovy [hunk 2]
```diff
@@ -508,9 +508,6 @@ def innerLoopNonWindowsOSs = ['Ubuntu15.10', 'Debian8.2', 'OSX', 'CentOS7.1', 'O
             // Archive the results
             Utilities.addArchival(newBuildJob, "bin/build.pack,bin/osGroup.AnyCPU.${configurationGroup}/**,bin/ref/**,bin/packages/**,msbuild.log")
 
-            // Use Server GC for Ubuntu/OSX Debug PR build & test
-            def useServerGC = ((os == 'Ubuntu15.10' || os == 'OSX') && configurationGroup == 'Release' && isPR)
```
Comment (mellinoe) [inline netci.groovy]: Do you need to change the call a little bit below this? It uses the "useServerGC" variable. Line 525

**Comment by Priya91** [comment]
@dotnet-bot test ci please

File: netci.groovy (hunk 2 above)
Comment (Priya91) [inline netci.groovy]: Made the call to pass false for this..

**Comment by Priya91** [comment]
@dotnet-bot test ci please

**Comment by Priya91** [comment]
@mmitche run triggers not working here..

**Comment by mmitche** [comment]
@Priya91 I believe we are hitting the rate limit.  It will eventually reset and things will show up.

**Comment by mmitche** [comment]
woops, hit close by accident

**Comment by Priya91** [comment]
Okie, thanks!

**Comment by Priya91** [comment]
@dotnet-bot test ci please

**Comment by weshaggard** [comment]
Thanks @Priya91. Is there any reason we cannot go ahead and move all the others to using build.sh as well?

cc @joperezr

**Comment by Priya91** [comment]
> Thanks @Priya91. Is there any reason we cannot go ahead and move all the others to using build.sh as well?

Do we have the dotnet cli packages for other rids, that's the only blocker i can think of..

**Comment by Priya91** [comment]
Verified the build definitions. Merging.

**Comment by weshaggard** [comment]
>Do we have the dotnet cli packages for other rids, that's the only blocker i can think of..

I don't think we have for all of them but I'm sure we have it for some more of them.

**Comment by joperezr** [comment]
> I don't think we have for all of them but I'm sure we have it for some more of them

Agreed, I believe at least CentOS should be fine. What I do know for a fact, is that these two are not supported yet: 'OpenSUSE13.2', 'RHEL7.2'

**Comment by weshaggard** [comment]
Looking at https://github.com/dotnet/cli it does appear as though RHEL7.2 is supported as well.

**Pull request merged**